packages/components/src/styles/interactive/interactive.stories.ts

```diff
@@ -55,6 +55,27 @@ export const Disabled = {
   }
```

Comment:
this should be in another PR as we use squash-merge in our PRs. This would result in one single commit with changes in 2 different components...

As other teams rely on our changelog, we need to be precise here.